PhotoMap/Features/Timeline/ViewModels/TimelineViewModel.swift

```diff
@@ -88,8 +108,24 @@ class TimelineViewModel: TimelineViewModelType {
                 groupedMarkers[marker.date.monthAndYear] = [marker]
             }
         }
-        self.markers = groupedMarkers
-        self.headerTitles = [String](groupedMarkers.keys).sorted { $0.toMonthAndYearDate > $1.toMonthAndYearDate }
+        let titles = [String](groupedMarkers.keys).sorted { $0.toMonthAndYearDate > $1.toMonthAndYearDate }
+        return (groupedMarkers, titles)
+    }
+    
+    private func searchMarkersWith(hashtag: String) {
+        let allmarkers = self.markers.values.flatMap { $0 }
+        var filteredMarkers = [Marker]()
+        
+        markersLoop: for marker in allmarkers {
+            for tag in marker.hashtags where tag.lowercased().contains(hashtag.lowercased()) {
+                filteredMarkers.append(marker)
+                continue markersLoop
+                
+            }
+        }
```

Comment:
Why not just:

```suggestion
        for marker in allmarkers {
            for tag in marker.hashtags where tag.lowercased().contains(hashtag.lowercased()) {
                filteredMarkers.append(marker)
            }
        }
```
?